docs/guide-for-application-teams.md

```diff
@@ -43,7 +43,14 @@ Adding Helm Charts
     -   Setup an [`ImageRepository`](https://fluxcd.io/docs/components/image/imagerepositories/) to query container registry for tags
     -   Setup an [`ImagePolicy`](https://fluxcd.io/docs/components/image/imagepolicies/) to choose what the latest tag is
     -   Setup an [`ImageUpdateAutomation`](https://fluxcd.io/docs/components/image/imageupdateautomations/) to describe which `GitRepository` object you want flux to update, and which directory
-    -   Add a [policy marker](https://fluxcd.io/docs/guides/image-update/#configure-image-update-for-custom-resources) to tell Flux how to update te yaml files
+    -   Add a [policy marker](https://fluxcd.io/docs/guides/image-update/#configure-image-update-for-custom-resources) to tell Flux how to update the yaml files
+  
+Managing persistence
+--------------------
+
+- `PersistentVolumes` will automatically fulfill the `PersistentVolumeClaims` created by the respective application teams.
+  - `storageClass` needs to be specified for all `PersistentVolumeClaims`.
```

Comment:
We're not sure which `storageClass` to choose. We noticed there is a `sciety-gp3` listed , we're not sure what it's for.

Reply:
After platform team advice, the Sciety team should use the team-specific `storageClass` (`sciety-gp3`) to allow cost tracking.